config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml

```diff
@@ -33,6 +33,21 @@ presubmits:
         - make
         args:
         - test-integration
+  - name: pull-kueue-test-e2e-main
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/kueue
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: pull-kueue-test-e2e-main
+      description: "Run kueue end to end tests"
+    spec:
+      containers:
+      - image: golang:1.18
+        command:
+        - make
+        args:
+        - test-e2e
```

Comment:
better have the kind create and delete here, rather than as part of test-e2e

Reply:
We decided to have this logic in our script.